website/docs/proto/config.mdx

```diff
@@ -255,6 +255,8 @@ As an example, let's configure [Node.js](https://github.com/moonrepo/node-plugin
 identifier).
 
 ```toml title=".prototools"
+npm = "bundler" # use bundled npm instead of specific version
```

Comment:
@iambpn Needs to be "bundled"

Reply:
Oops sorry about that 😬. I'll correct this.